client/state/console-dispatch/README.md

```diff
@@ -0,0 +1,65 @@
+# Console Dispatcher
+**A Redux [store enhancer](https://github.com/reactjs/redux/blob/master/docs/Glossary.md)**
```

Comment:
Maybe link directly to: https://github.com/reactjs/redux/blob/master/docs/Glossary.md#store-enhancer